lib/board.rb

```diff
@@ -1,11 +1,5 @@
-require_relative 'board'
```

Comment:
Don't know why this commit is on this branch, when I committed and pushed to #16...